conans/tools.py

```diff
@@ -94,6 +94,7 @@ def get(*args, **kwargs):
 which = tools_files.which
 unix2dos = tools_files.unix2dos
 dos2unix = tools_files.dos2unix
+valid_lib_extensions = tools_files.valid_lib_extensions
```

Comment:
No, certainly we don't want to expose it for usage by recipes.
By default we don't expose anything to tools, unless it has been explicitly requested and approved.

Reply:
So I need to import as "private"

Reply:
No, just do not import it from tools. It is not a tool, it is for internal usage only.